docs/design/job-scale-up-down.md

```diff
@@ -0,0 +1,103 @@
+# Volcano Job scale up and down
```

Comment:
I assume the scope of this proposal is to make sure controller can response to the job scale up and down. no scale up/down decision need to be made here.

Reply:
Yeah, correct.